docs/tutorial.md

```diff
@@ -163,7 +163,10 @@ There are also operators for getting the parents (`foo-`), children (`foo+`),
 ancestors (`::foo`), descendants (`foo::`), DAG range (`foo::bar`, like
 `git log --ancestry-path`), range (`foo..bar`, same as Git's). There are also a
 few more functions, such as `heads(<set>)`, which filters out revisions in the
-input set if they're ancestors of other revisions in the set.
+input set if they're ancestors of other revisions in the set, and
+`description(foo)`, the commits whose description (commit message) contains `foo`.
```

Comment:
I think we can remove the mention of `heads()` here. It's mentioned here for historical reasons (from when `jj log`'s default revset was too large and we told the user to define an alias using `heads()` here).

Reply:
I pushed a new commit to remove the mention of `heads(x)`.